ivy/backend_handler.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 from ivy import verbosity
 from typing import Optional
+import gc
```

Comment:
I think this is another unused import that can be removed

Reply:
Are you referring to the import of `gc` module? It's basically being used [here](https://github.com/unifyai/ivy/blob/0c8d88b6d238509033b3298966e1450e506d4c3e/ivy/backend_handler.py#L273)

Reply:
Oh right, yeah thanks for clearing it out!